src/components/Signup/Signup.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FormattedHTMLMessage, injectIntl } from 'react-intl';
+import { timeout } from '../../utils';
+import { Formik, Form } from 'formik';
+import { FieldGroup, InputFieldItem, CheckboxFieldItem } from '../form-helpers/form-helpers';
+
+const fieldNames = {
+  firstname: 'firstname',
+  lastname: 'lastname',
+  phone: 'phone',
+  email: 'email',
+  password: 'password',
+  accept_privacy_policies: 'accept_privacy_policies',
+  accept_promotions: 'accept_promotions',
+};
+
+/** Prepare empty values for all fields
+ * It is required because in another way, the fields are not marked as touched.
+ */
+const getFormInitialValues = () =>
+  Object.keys(fieldNames).reduce(
+    (accumulator, currentValue) => ({ ...accumulator, [currentValue]: '' }),
+    {},
+  );
+
+export default injectIntl(function({ intl }) {
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const validate = (values) => {
+    const errors = {};
+    if (!values[fieldNames.firstname]) {
+      errors[fieldNames.firstname] = _('validation_messages.error_required_field');
+    }
+
+    if (!values[fieldNames.lastname]) {
+      errors[fieldNames.lastname] = _('validation_messages.error_required_field');
+    }
+
+    if (!values[fieldNames.phone]) {
+      errors[fieldNames.phone] = _('validation_messages.error_required_field');
+    } else {
+      // TODO: validate phone
```

Comment:
maybe can use REGEX from doppler app phone validation

Reply:
No, Doppler uses three fields in place of one.

On the other hand, we want to use the same library that we are already using in sites to validate phone numbers, so, I think that it could be done as a different task.

Reply:
oh i didn't know in sites already using one

Reply:
Oh, yes, and it is great!. @IgnacioRodrigues do you have a link to try the phone validation?

Reply:
Yes, you can look at the following form in the field phone https://m.fromdoppler.com/contacto/